<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmReportOptions</class>
 <widget class="QMainWindow" name="frmReportOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>359</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SWMM Report Options</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cmdOK">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdCancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxNode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Nodes</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdNodeAll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdNodeNone">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxReport">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Report Options</string>
    </property>
    <widget class="QCheckBox" name="cbxControls">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Controls</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxInput">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Summary</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxFlow">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Flow Stats</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbxContinuity">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Continuity Checks</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxLinks">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Links</string>
    </property>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdLinksAll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdLinksNone">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxSubcatchments">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Subcatchments</string>
    </property>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdSubcatchmentsAll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdSubcatchmentsNone">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
